Src/PerformanceCollector/NetCore/Stubs/Implementation/QuickPulse/Process/QuickPulseTopCpuCollector.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using System;
+    using System.Collections.Generic;
+
+    using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse.Helpers;
+
+    /// <summary>
+    /// Top CPU collector.
+    /// </summary>
+    internal sealed class QuickPulseTopCpuCollector : IQuickPulseTopCpuCollector
+    {
+        private static readonly Tuple<string, int>[] emptyResult = new Tuple<string, int>[0];
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="QuickPulseTopCpuCollector"/> class. 
+        /// </summary>
+        /// <param name="timeProvider">Time provider.</param>
+        /// <param name="processProvider">Process provider.</param>
+        public QuickPulseTopCpuCollector(Clock timeProvider, IQuickPulseProcessProvider processProvider)
+        {
+        }
+
+        /// <summary>
+        /// Gets a value indicating whether the initialization has failed.
+        /// </summary>
+        public bool InitializationFailed { get; } = false;
+
+        /// <summary>
+        /// Gets a value indicating whether the Access Denied error has taken place.
+        /// </summary>
+        public bool AccessDenied { get; } = false;
+
+        /// <summary>
+        /// Gets top N processes by CPU consumption.
+        /// </summary>
+        /// <param name="topN">Top N processes.</param>
+        /// <returns>List of top processes by CPU consumption.</returns>
+        public IEnumerable<Tuple<string, int>> GetTopProcessesByCpu(int topN)
+        {
+            return QuickPulseTopCpuCollector.emptyResult;
```

Comment:
As of now netstandard does not have API for reading PerformanceCounters, the issure requesting support is still open: https://github.com/dotnet/corefx/issues/3906, so for now Live Metrics will not be returning CPU/Memory/Performance Counters for .NET Core apps